tests/estimator/skinetic/skinetic_check.py

```diff
@@ -59,22 +60,36 @@ def grab_stat_entries(stat_file_name,name):
     # read stat.h5 for skinetic entry
     stat_entry = grab_stat_entries(fstat,est_name)[0] # only 1 skinetic
     h5_data = stat_entry['value']
+
+    passed = True
     
+    # test h5 entries against scalar.dat entries
     if not np.allclose(h5_data,data[:,skinetic_idx]):
         print "species kinetic energy estimator h5 test failed"
```

Comment:
Could you make this message more specific - something like "Species kinetic energy estimator failed - the values in the HDF file do not match the values in the .scalar.dat file"
(and same with the other message below)